custom-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -0,0 +1,421 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tikit",
+    "description": "This is a ticketing management system.",
+    "version": "1.0",
+    "contact": {
+      "name": "Celebaltech Support",
+      "url": "https://www.celebaltech.com/support",
+      "email": "support@celebaltech.com"
+    }
+  },
+  "host": "tikit.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.celebaltech.com"
+    },
+    {
+      "propertyName": "Privacy Policy",
+      "propertyValue": "https://www.celebaltech.com/privacypolicy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;Sales and CRM"
+    }
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/Ticket/AddTicket": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@1MrSharma Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
Updated the changes required.